packages/backend/src/modules/task/task.controller.ts

```diff
@@ -32,6 +32,14 @@ class TaskController {
     const taskDto: TaskDTO = this._taskMapper.mapToDto(task)
     return res.status(201).send(taskDto)
   }
+
+  async findByIdAndRemove(req: express.Request, res: express.Response) {
+    const task: Task = await this._taskService.findByIdAndRemove(req.params.taskId)
+    if (!task) {
+      return res.status(404).send('Task not found.')
+    }
+    return res.status(201).send('Task has been successfully removed.')
```

Comment:
200 😄 201 is for POST requests only

Reply:
![image](https://user-images.githubusercontent.com/80964094/166979266-4c622a2d-d600-416b-ad79-b6d9edd6f4e1.png)

Reply:
thanks, done